js/src/common/models/User.js

```diff
@@ -29,9 +29,14 @@ Object.assign(User.prototype, {
   discussionCount: Model.attribute('discussionCount'),
   commentCount: Model.attribute('commentCount'),
 
-  canEdit: Model.attribute('canEdit'),
+  canEditCredentials: Model.attribute('canEditCredentials'),
+  canEditGroups: Model.attribute('canEditGroups'),
   canDelete: Model.attribute('canDelete'),
 
+  canEdit: computed('canEditCredentials', 'canEditGroups', 'canEdit', function (canEdit, canEditCredentials, canEditGroups) {
```

Comment:
`computer` will try to call its dependencies as methods, so it'll try to call `canEdit` and end up in a recursive loop. Maybe we can set it to `canEditAttributes` on the js user model?